src/correct-round.rkt

```diff
@@ -262,24 +275,23 @@
      ; when output crosses zero and x is negative - means that y was fractional and not fixed
      ; solution - add more slack for y to converge
      (define y-slack (if (and (not (equal? (mpfr-sign outlo) (mpfr-sign outhi)))
-                              (bfnegative? (ival-lo x))
-                              (bfnegative? (ival-hi x)))
+                              (bfnegative? (ival-lo x)))
                          (get-slack)
                          0))
 
-     (define xlo-exp (mpfr-exp (ival-lo x)))
-     (define xhi-exp (mpfr-exp (ival-hi x)))
-     (define x-slack ; it is likely to be a handling case from IEEE, x is not close enough
+     #;(define xlo-exp (mpfr-exp (ival-lo x)))
+     #;(define xhi-exp (mpfr-exp (ival-hi x)))
+     #;(define x-slack ; it is likely to be a handling case from IEEE, x is not close enough
        (if (or (equal? xlo-exp -9223372036854775807)
-               (equal? xhi-exp -9223372036854775807)  ; if interval contains 0
-               (and (> 2 xlo-exp) (<= 2 xhi-exp)))    ; if interval crosses 1
+               (equal? xhi-exp -9223372036854775807)        ; if interval contains 0
+               (and (equal? 1 xlo-exp) (equal? 1 xhi-exp))) ; if interval possibly contains 1
            (get-slack)
            0))
```

Comment:
This is all commented out? Why?

Reply:
I just feel that it doesn't matter how close the interval will be if it is a handling case from IEEE.
It always will be 1 ulp distance between lo/hi in inputs. 
And since MPFR seems to specifically match `0.bf` - then it doesn't matter how close the interval will be, even `+min.bf` will be treated not as `0.bf`. I decided to leave it, maybe later can be useful

Reply:
Ok.